src/components/basic/DlAlert/DlAlert.vue

```diff
@@ -111,6 +121,16 @@ export default defineComponent({
         text: {
             type: String,
             default: null
+        },
+        darkMode: {
```

Comment:
this is an incorrect approach in this case. adding a prop to define theme mode is not necessary.

Reply:
this is not a definition, this is turning on and off the dark mode for the alert
alert is used in toast message
in toast message colors should not change in dark mode, but in alert should